### DISCUSSION TITLE
Add Search FIlter in game by year that beatmap created

### DISCUSSION THREAD

**Discussion by alexoshowo** [ppy/osu#23738] [Ideas]
Like you can filter by writing in search "creator=sotarks", you could search it by year that beatmap created

**Comment by Rekunan** [comment]
I think this would need implementation on https://github.com/ppy/osu-web first, so I recommend making an issue over there for this.

For now, though, there is `ranked=` which works similarly, except by when the beatmap was ranked instead of created.
